CONTRIBUTING.md

```diff
@@ -37,4 +37,33 @@ W skrócie:
 * Linię z tytułem zaczynamy Wielką literą
 * Tytuł, tak jak w podstawówce uczono, nie posiada na końcu kropki
 * Tytuł powinien być zapisany z wykorzystaniem bezokoliczników (używamy form: REFACTOR, UPDATE, REMOVE, RELEASE, MERGE, FIX)
-* Użyj ciała commita, aby wyjaśnić WWH - What Why How
\ No newline at end of file
+* Użyj ciała commita, aby wyjaśnić WWH - What Why How
+
+
+# Kilka drobnych wskazówek:
+Te wskazówki przydadzą się również osobom zatwierdzającym zmiany.
+
+### Etykiety:
+* **"Assigned"** - oznacza, że Twojoje isse/PR zostało zatwierdzone, a Ty wiesz, że jest sens pracować nad tym dalej.
+* **"Bug"** - dostaje issue, które dotyczy jakiegoś błędu w kodzie
+* **"Dublicate"** - należy dodać, oraz zamknąć issue linkując do poprzedniego.
+* **"Feature"** - Jest to jakiś nowy ficzer.
+* **"Need Review"** - Skonczyłeś zmiany, czekaj na CodeReview.
+* **"Question"** - Jeżeli Twooje issue to pytanie 
```

Comment:
Poprawię jak znajdę troszkę czasu, dzięki.

hmm może takie coś być?

W dniu 11 lipca 2016 11:40 użytkownik Arkadiusz Waluk <
notifications@github.com> napisał:

> In CONTRIBUTING.md
> <https://github.com/CodersCommunity/forum.pasja-informatyki.local/pull/28#discussion_r70228114>
> :
>
> > \ No newline at end of file
> > +* Użyj ciała commita, aby wyjaśnić WWH - What Why How
> > +
> > +
> > +# Kilka drobnych wskazówek:
> > +Te wskazówki przydadzą się również osobom zatwierdzającym zmiany.
> > +
> > +### Etykiety:
> > +* **"Assigned"** - oznacza, że Twojoje isse/PR zostało zatwierdzone, a Ty wiesz, że jest sens pracować nad tym dalej.
> > +* **"Bug"** - dostaje issue, które dotyczy jakiegoś błędu w kodzie
> > +* **"Dublicate"** - należy dodać, oraz zamknąć issue linkując do poprzedniego.
> > +* **"Feature"** - Jest to jakiś nowy ficzer.
> > +* **"Need Review"** - Skonczyłeś zmiany, czekaj na CodeReview.
> > +* **"Question"** - Jeżeli Twooje issue to pytanie
>
> Twooje
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CodersCommunity/forum.pasja-informatyki.local/pull/28/files/98f764bbbfd9377e6e22de9be8c5a67668161113#r70228114>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ANOczS1T6K4AuNMEazcWSI9M474oPWnRks5qUg-HgaJpZM4JJJuN>
> .
>